src/applications/personalization/account/components/ConnectedAccountsSection.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import isBrandConsolidationEnabled from '../../../../platform/brand-consolidation/feature-flag';
```

Comment:
We no longer need to use this utility. It’s VA.gov all the time now. Moving forward if you come across usage of this utility in files you are changing feel free to pull its usage out.